common/src/main/java/org/mvndaemon/mvnd/common/Environment.java

```diff
@@ -68,14 +68,18 @@ public String asCommandLineProperty(String value) {
      * The number of log lines to display for each Maven module that is built in parallel.
      */
     MVND_ROLLING_WINDOW_SIZE("mvnd.rollingWindowSize", null, "0", false),
+    /**
+     * The automatic log purge period
```

Comment:
A short description of the supported syntax and a couple of duration examples would be nice here and on other Duration options. 

We should slowly start adding `@since` so that users see how old is which option.

Reply:
Makes sense, I'll add one.
I think we can start using `@since` when we'll switch to a non `0.0.x` version, which kinda assumes, very early code and non stabilised api.